testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -676,12 +672,26 @@ module('Keyboard navigation', setupModule, () => {
 
     test('enter should clear search value', function(assert) {
         this.keyboard.type('1');
+
         assert.strictEqual(this.instance.option('text'), 'January 10 2012', 'text has been changed');
+        assert.strictEqual(this.instance._searchValue, '1', 'there is searchValue');
```

Comment:
cannot we use input val instead of private property?

Reply:
thought that better use private property if it make test more obvious
ok then, I returned the test like it was except of section moving part on enter key